### PR TITLE
fix(nasha): snapshot policies limit to 10

### PR DESCRIPTION
**Pull request by ghyenne**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/nasha-refresh` 
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | MANAGER-9853 
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

## Description
I add a rule of snapshot policies limit to 10

## Related
:flags:  Translation:

- [ ] TRANS-45604

### CODE REVIEW

File: packages/manager/modules/nasha/src/dashboard/partition/snapshots/snapshots.template.html [hunk 1]
```diff
@@ -4,10 +4,23 @@
...
     <h4 data-translate="nasha_dashboard_partition_snapshots_subtitle"></h4>
+    <p class="d-flex flex-column">
+        <strong
+            data-translate="nasha_dashboard_partition_snapshots_count_title"
+        ></strong>
+        <span
+            data-translate="nasha_dashboard_partition_snapshots_count"
+            data-translate-values="{
+                count: $ctrl.customSnapshots.length,
+                max: $ctrl.MAX_CUSTOM_SNAPSHOT,
```
Comment (sachinrameshn) [inline packages/manager/modules/nasha/src/dashboard/partition/snapshots/snapshots.template.html:15]: Suggestion: Can we move the calculation {{ count }}/{{ max }}to controller ? 
  data-translate-values="{
                maximum: $ctrl.maximum,}"

File: packages/manager/modules/nasha/src/dashboard/partition/snapshots/snapshots.controller.js [hunk 2]
```diff
@@ -28,6 +29,8 @@ export default class NashaDashboardPartitionSnapshotsController {
     this.customSnapshotNamePattern = CUSTOM_SNAPSHOT_NAME_PATTERN;
     this.customSnapshotNamePrefix = CUSTOM_SNAPSHOT_NAME_PREFIX;
     this.customSnapshotNameSeparator = CUSTOM_SNAPSHOT_NAME_SEPARATOR;
+
+    this.MAX_CUSTOM_SNAPSHOT = MAX_CUSTOM_SNAPSHOT;
```
Comment (sachinrameshn) [inline packages/manager/modules/nasha/src/dashboard/partition/snapshots/snapshots.controller.js:33]: Suggestion: Can we move the calculation {{ count }}/{{ max }}to controller ? 
 this.maximum = this.customSnapshots.length / MAX_CUSTOM_SNAPSHOT;

File: packages/manager/modules/nasha/src/dashboard/partition/snapshots/translations/Messages_fr_FR.json [hunk 3]
```diff
@@ -15,5 +15,7 @@
   "nasha_dashboard_partition_snapshots_reset_snapshot_types": "Supprimer les changements de fréquences",
   "nasha_dashboard_partition_snapshots_create_rule": "Sont autorisés : caractères alphanumériques, point, deux points et tiret",
   "nasha_dashboard_partition_snapshots_warning_1": "20% d'espace additionnel vous est fourni pour vos snapshots.",
-  "nasha_dashboard_partition_snapshots_warning_2": "Dans le cas où vous dépasseriez cet espace, les snapshots seront stockés sur votre espace de stockage primaire."
+  "nasha_dashboard_partition_snapshots_warning_2": "Dans le cas où vous dépasseriez cet espace, les snapshots seront stockés sur votre espace de stockage primaire.",
+  "nasha_dashboard_partition_snapshots_count_title": "Nombre de snapshots",
+  "nasha_dashboard_partition_snapshots_count": "{{ count }}/{{ max }} (maximum)"
```
Comment (sachinrameshn) [inline packages/manager/modules/nasha/src/dashboard/partition/snapshots/translations/Messages_fr_FR.json:20]: Suggestion: Can we move the calculation {{ count }}/{{ max }}to controller ?
"nasha_dashboard_partition_snapshots_count": "{{ maximum }} (maximum)"

File: packages/manager/modules/nasha/src/dashboard/partition/snapshots/snapshots.template.html (hunk 1 above)
Comment (ghyenne) [reply packages/manager/modules/nasha/src/dashboard/partition/snapshots/snapshots.template.html:15]: This is not a calculation but a display of information. Would you prefer I use this.$translate.instant()?

File: packages/manager/modules/nasha/src/dashboard/partition/snapshots/snapshots.controller.js (hunk 2 above)
Comment (ghyenne) [reply packages/manager/modules/nasha/src/dashboard/partition/snapshots/snapshots.controller.js:33]: This is not a calculation but a display of information. Would you prefer I use this.$translate.instant()?

File: packages/manager/modules/nasha/src/dashboard/partition/snapshots/translations/Messages_fr_FR.json (hunk 3 above)
Comment (ghyenne) [reply packages/manager/modules/nasha/src/dashboard/partition/snapshots/translations/Messages_fr_FR.json:20]: This is not a calculation but a display of information. Would you prefer I use this.$translate.instant()?
count_snapshots

Comment (sachinrameshn) [reply packages/manager/modules/nasha/src/dashboard/partition/snapshots/translations/Messages_fr_FR.json:20]: Okay Guillaume you can ignore my comments

**Review by sachinrameshn — DISMISSED** [review]

File: packages/manager/modules/nasha/src/dashboard/partition/snapshots/translations/Messages_fr_FR.json (hunk 3 above)
Comment (anooparveti) [reply packages/manager/modules/nasha/src/dashboard/partition/snapshots/translations/Messages_fr_FR.json:20]: @ghyenne , it is better if you maintain only `(maximum)` in the trans file.

Comment (ghyenne) [reply packages/manager/modules/nasha/src/dashboard/partition/snapshots/translations/Messages_fr_FR.json]: I created a variable in controller:

```
this.snapshotsCount = `${
      this.customSnapshots.length
    }/${MAX_CUSTOM_SNAPSHOT} (${this.$translate.instant(
      'nasha_dashboard_partition_snapshots_count_max',
    )})`;
 ```

**Review by anooparveti — APPROVED** [review]

**Review by JeremyDec — CHANGES_REQUESTED** [review]
Hello @ghyenne, could you please resolve existing conflicts?

**Review by anooparveti — APPROVED** [review]

**Review by anooparveti — APPROVED** [review]

**Review by sachinrameshn — APPROVED** [review]

**Review by vks-ovh — APPROVED** [review]

**Pull request merged**